src/Entity/Homepage.php

```diff
@@ -81,6 +60,9 @@ public function __construct()
         $this->initializeChannelsCollection();
     }
 
+    /**
+     * @return int|null
+     */
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏do we have to add the phpdoc? Because it does not add anything more than the definition of the method

Reply:
@delyriand Since we don't have the definition of the properties directly in the class I think it's nice to keep this phpdoc instead of having to go check the xml file.
What do you think ?